packages/contentful--create-contentful-app/src/app-actions/typescript/index.ts

```diff
@@ -0,0 +1,15 @@
+// import type { APIGatewayEvent, Context } from 'aws-lambda';
+
+exports.handler = async (event: any) => {
+  console.debug(`Event: ${JSON.stringify(event, null, 2)}`);
```

Comment:
Why console here?

Reply:
I stole this from merge. But, I suppose the user wouldn't have a way of looking at this console anyway. I'll remove it.